libmscore/cmd.cpp

```diff
@@ -1067,8 +1067,11 @@ bool Score::makeGap1(const Fraction& baseTick, int staffIdx, const Fraction& len
 
             if (newLen > Fraction(0,1)) {
                   const Fraction endTick = tick + newLen;
-                  deleteAnnotationsFromRange(tick2rightSegment(tick), tick2rightSegment(endTick), track, track + 1, selectionFilter());
-                  deleteSpannersFromRange(tick, endTick, track, track + 1, selectionFilter());
+                  typedef SelectionFilterType Sel;
```

Comment:
We don't usually do `typedef` inside code blocks... And even if you do, please name it as "SFT" or alike so that it can be clearer.

Reply:
I've seen this being done in other places

Reply:
Not normally indeed, but see https://github.com/musescore/MuseScore/blame/66099812d88fd4d32ad202ab00d8b064fdd489bc/libmscore/cmd.cpp#L1030.  This is basically a direct copy of that that, so best to keep it consistent I think.

Reply:
Here though is it done to shorten one other statement, and for that it isn't worth the efoort IHMO

Reply:
As a matter of fact, this code is straight copy as was implemented in Score::makeGap() (https://github.com/musescore/MuseScore/pull/4855). All I did was just copying, not that much creativity :-)

Reply:
@MarcSabatella That's exactly what I did.

Reply:
Oh, you're right, I missed that.